farsifonts.sh

```diff
@@ -7,25 +7,21 @@
 # You can use "axel" instead of default "wget", just install axel first and call this
 # with axel parameter.
 
-FONT_PATH="$HOME/.fonts/truetype/"
-function detect(){
-  type -P $1 >/dev/null  || { echo "Require $1 but not installed. Aborting." >&2; exit 1; }
-}
-
+FONT_PATH="$HOME/.local/share/fonts/"
```

Comment:
the `.local` folder is somehow debian specific. other distros might need some configuration change to use this directory. I will look into it later